### DISCUSSION TITLE
updateVirtualSelect not working?

### DISCUSSION THREAD

**Discussion by jgsarm-rb** [dreamRs/shinyvs#2] [Q&A] [answered]
Hi there! So i've been testing for a little bit now, and one issue I've encountered so far is in the function updateVirtualSelect(). 

When virtual select updates, nothing is selected even though `selected = ...` is specified.

**Comment by pvictor** [comment, marked as answer]
There's was indeed a problem, selected value was set before updating new choices.
Should work as expected now

**Comment by jgsarm-rb** [comment]
Can confirm that it is working now
